istio/manifest.json

```diff
@@ -46,7 +46,10 @@
       },
       "service_checks": {
         "metadata_path": "assets/service_checks.json"
-      }
+      },
+      "process_signatures": [
+        "istio"
+      ]
```

Comment:
Could you revert this change since it belongs to your other PR? Thank you 🙇

Reply:
Done!